doniclient/v1/client.py

```diff
@@ -32,6 +32,9 @@ def get_by_uuid(self, uuid):
         except json.JSONDecodeError:
             return resp
 
+    def get(self, name_or_uuid):
```

Comment:
interesting, this is a standard function understood by osc, which is different than get_by_uuid?

Reply:
I couldn't find explicit docs stating the requirements at https://docs.openstack.org/python-openstackclient/latest/contributor/plugins.html.

Generally, osc_lib seems to fall back on direct REST calls, but there are portions of the utilities that call `manager.get()` `manager.list()`, `manager.find()`, etc. so there seems to be a convention of sorts.
https://github.com/openstack/osc-lib/blob/master/osc_lib/utils/__init__.py